src/sass/accordion/accordion.scss

```diff
@@ -25,6 +25,13 @@ ul.accordion summary.details__summary {
     justify-content: space-between;
     min-height: 48px;
     padding: 12px 16px;
+
+    span.details__icon svg {
```

Comment:
Can we do 
`ul.accordion summary.details__summary ul.accordion summary.details__summary {` instead of nesting this?
While its longer to read and type out its also easier to debug